Library/Homebrew/extend/ENV/super.rb

```diff
@@ -332,13 +332,16 @@ def refurbish_args
     end
   end
 
+  def x11_if_installed
+  end
+
   # @private
   def noop(*_args); end
   noops = []
 
   # These methods are no longer necessary under superenv, but are needed to
   # maintain an interface compatible with stdenv.
-  noops.concat %w[fast O4 Og libxml2 set_cpu_flags macosxsdk remove_macosxsdk]
+  noops.concat %w[fast O4 Og libxml2 set_cpu_flags]
```

Comment:
Since you're touching this code here and I think this is pretty obscure (and not very accessible to tooling), would you mind replacing this with a series of `alias_method <whatever>, :noop`?